### PR TITLE
IPv6Network: fix netmask equality check

### PR DESCRIPTION
**Pull request by Lctrs**

### CODE REVIEW

**Comment by tux-rampage** [comment]
Thank you very much for your contribution. I'll review this later for the failing build. Maybe it's not related to this change.

**Comment by Lctrs** [comment]
It's not related. The `equals()` methods are not covered by unit tests. Build failures are due to composer v2 being the default on Travis now.

BTW, would you be interested in a migration to Github Actions ? I can work on that.

**Review by tux-rampage — CHANGES_REQUESTED** [review]
@Lctrs sorry for the delay. I wasn't able to respond sooner.
Would you mind adding a test to cover this?

**Comment by Lctrs** [comment]
@tux-rampage Done :)

File: .travis.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ script:
   - composer run-script test -- --colors=always --coverage-clover ./coverage/clover.xml --coverage-xml=./coverage/coverage-xml --log-junit=./coverage/junit.xml
   - composer run-script analyse -- --no-progress
   - composer run-script cs-check
-  - infection --min-msi=75 --min-covered-msi=80 --threads=$(nproc) --coverage=coverage
+  - infection --min-msi=84 --min-covered-msi=86 --threads=$(nproc) --coverage=coverage
```
Comment (tux-rampage) [inline .travis.yml:34]: 👍🏻

**Review by tux-rampage — APPROVED** [review]
Perfect, thanks 👍🏻

**Comment by tux-rampage** [comment]
This was merged into `develop`. I just forgot to change the target branch here, so GitHub didn't recognize it.

**Comment by Lctrs** [comment]
@tux-rampage Thanks !

I saw that you introduced GHA in the project, but it fails at the moment. My invitation to work on it still remains, it will be based on https://github.com/ergebnis/php-library-template.

**Comment by tux-rampage** [comment]
Thanks, feel free to open a PR to complete GitHub actions. The one right here seem to come from Travis after I enabledthis feature.

**Pull request closed without merging**